package.json

```diff
@@ -64,6 +64,7 @@
     "grunt-jscs": "2.1.0",
     "grunt-mocha-cli": "1.14.0",
     "hawk": "3.1.0",
+    "halfpenny": "*",
```

Comment:
This idea also breaks away from our current gitflow process where features are merged into the develop branch, then release, and finally master. In the above scenario, we would merge changes to halfpenny directly into master, then publish on NPM ASAP. If the changes were merged into develop, we would need to publish from the develop branch which I have no experience with, but seems dodgy...